scripts/figures/zones.R

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8-unix -*-
+library(here)
+library(tidyverse)
+
+read_tsv_helmet <- function(..., comment = "#") {
+  readr::read_tsv(..., comment = comment) %>%
+    dplyr::rename(zone = X1) %>%
```

Comment:
So I guess this fixes the problem in https://github.com/HSLdevcom/helmet-model-system/issues/325? Do you still think we should fix this on model-system side?

Reply:
Yes, nice catch! :) I created this workaround to use the current data format more easily. However, I'd prefer it being fixed in `helmet-model-system` because I do not want to depend on the autogenerated column name "X1". Different applications autogenerate different names. This is from QGIS where I joined `car_density.txt` to `sijoittelualueet2019.shp` and the first column is now `field_1`:
![kuva](https://user-images.githubusercontent.com/20718745/121692166-7d5a2480-cad0-11eb-95b6-ea8d1ffa496e.png)